### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by s-rathi**
### :pencil: This pull request contains the README.md file 

**Type:** :memo: `documentation` 

The README.md file contains information about the original source of atomic data stored in the `carsus-data-cmfgen` repository.

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# carsus-data-cmfgen
+`carsus-data-cmfgen` mirrors the atomic data hosted on John Hillier's website https://sites.pitt.edu/~hillier/web/CMFGEN.htm
+ 
+**Note:** If anyone is using this data, then please cite the original database.
+To know more about the cmfgen, please refer to https://ui.adsabs.harvard.edu/abs/2011Ap%26SS.336...87H/abstract.
```
Comment (chvogl) [inline README.md:5]:
```suggestion
To learn more about the CMFGEN atomic data, please refer to https://ui.adsabs.harvard.edu/abs/2011Ap%26SS.336...87H/abstract.
```

**Review by chvogl — APPROVED** [review]
Looks good! I would maybe slightly rephrase the last sentence.

**Review by atharva-2001 — APPROVED** [review]
I think this can be merged once @chvogl's suggested changes are in.

**Comment by andrewfullard** [comment]
Woops, jumped the gun there

**Pull request merged**